doc/htmldoc/networks/scripts/connections.py

```diff
@@ -21,8 +21,8 @@
 
 # create connectivity figures for spatial manual
 
-import nest
 import matplotlib.pyplot as plt
+import nest
```

Comment:
I am not really sure if I like it that NEST is hidden here between matplotlib and numpy.

Is there any way to configure isort so that nest always comes last?

Reply:
@heplesser Yes, the default sorting scheme of `isort` is actually as you describe with 1. Python library, 2. third-party and 3. first-party (in this case `nest`).

The reason `nest` is sorted as third-party is because I declared it to be counted as that. The rationale for that decision was that `nest` will then be sorted correctly for users copying our examples for their own use. 

I have no strong feelings about `nest` being counted as third- or first-party, so if want `nest` to be sorted as a first-party module I can fix that fairly quick.

Reply:
That is a good point. Maybe my view of the world is just too NEST-centric, seeing all other imports than `nest` to be just helpers in scripts importing `nest` as the main workhorse.  @terhorstd What do you think?

Reply:
Ok, I agree treating NEST as third party in example scripts makes sense.